Frontend/src/Components/Notifications/Notifications.jsx

```diff
@@ -24,13 +24,16 @@ const Notifications = () => {
     successMessage,
     errorMessage,
   } = useReview();
+  const [filterType, setFilterType] = useState("");
 
   useEffect(() => {
     startLoading();
-    if (notifications.length > 0) {
-      stopLoading();
-    }
-  }, [notifications]);
+    fetchNotifications(filterType).then(() => stopLoading());
+  }, [filterType]);
```

Comment:
i'm guessing this is called on first load of notifications? is there a way we can avoid fetching notifications twice (once here and once in socketcontext on line 39)